apps/frontend/app/(website)/blog/tag/[tag]/page.tsx

```diff
@@ -9,7 +9,7 @@ export async function generateStaticParams() {
   const tags = await client.fetch(groq`*[_type == "blog.tag"]`);
 
   const paths = tags.map((tag: any) => ({
-    tag: tag.slug.current,
+    tag: tag.slug?.current,
```

Comment:
@Hazetheai we had to edit blog file, because build was failing with "Cannot read current of undefined".  
<img width="846" alt="image" src="https://github.com/codemod-com/codemod/assets/125881252/8ffa87d6-3a02-4474-a460-d29837d8bb02">